.github/workflows/ci.yaml

```diff
@@ -563,18 +556,31 @@ jobs:
     runs-on: ${{ matrix.os.image }}
 
     steps:
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: "stable"
+      
       - uses: actions/checkout@v2
 
       - uses: dart-lang/setup-dart@v1
```

Comment:
Given we already have flutter, do we need to setup dart?

Reply:
indeed I didn't know [Flutter SDK comes with Dart SDK included](https://dart.dev/get-dart#:~:text=Note%3A%20The%20Flutter%20SDK%20includes,interface%20in%20its%20bin%20folder.) ! done in c8575ce64065d66ed7ae9ef2f171d322c5a02a9b